tests/golden/defaults/openshift4-logging/openshift4-logging/30_cluster_logging.yaml

```diff
@@ -13,15 +13,5 @@ spec:
   logStore:
     lokistack:
       name: loki
-    retentionPolicy:
```

Comment:
I assume this had no effect with Loki anyway?

Reply:
Yes, this is for Elasticsearch only.